components/x11/x11-protocols/history

```diff
@@ -0,0 +1,29 @@
+bigreqsproto@1.1.2-2018.0.0.0 x11/header/x11-protocols
```

Comment:
Don't change it in two places - either use meta-packages/history component or history file in x11-protocols.
Also, use full fmris.